.github/workflows/load-tests.yml

```diff
@@ -101,7 +107,12 @@ jobs:
             /home/runner/.cache/go-build
           key: v1-go-pkg-mod-${{ runner.os }}-${{ hashFiles('**/go.mod', '**/go.sum') }}
       - run: mkdir -p results && touch results/TESTRESULTS.md
-      - run: make otelcontribcol
+      - name: Download Collector Binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: collector-binaries
+          path: bin/
+      - run: chmod +x bin/*
```

Comment:
How come we need to make them executable here? Isn't already done as part of the build or is there something I am missing?

Reply:
[Load tests were failing](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/4143804143?check_suite_focus=true#step:13:104) after redownloading the binaries, due to them not being executable.

Reply:
upload-artifact appears to be breaking file permissions https://github.com/actions/upload-artifact/issues/38